packages/devtools/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 0.1.16 2020-02-12
+* Add zoomable timeline grid and timestamps to flame chart. (#1624)
+* update goldens (#1625)
```

Comment:
is it possible to special case commits about updating goldens and omit them? This can also be done manually after the changelog generation.

Reply:
Yes it is.  I've added an explicit filter.